modules/web-console/frontend/app/configuration/components/modal-preview-project/controller.ts

```diff
@@ -86,60 +86,61 @@ export default class ModalPreviewProjectController {
 
     doStuff(cluster: CluserLike, isDemo: boolean) {
         this.IgniteLoading.start('projectStructurePreview');
+
         return this.PageConfigure.getClusterConfiguration({clusterID: cluster.id, isDemo})
-        .then((data) => {
```

Comment:
These changes to ModalPreviewProjectController look unnecessary to me.

Reply:
Reverted